mandatory/2-function-creation.js

```diff
@@ -4,7 +4,13 @@
     1. the user should be 18 or older
     2. the user must be logged in
 */
-function isAcceptableUser(userAge, isLoggedIn) {}
+function isAcceptableUser(userAge, isLoggedIn) {
+  if(userAge >=18 && isLoggedIn){
+    return true
```

Comment:
let isAcceptable = userAge >=18 && isLoggedIn; inside of the isAcceptableUser function. No need for if and else function.
return isAccptable; 
Here should be just return isAccptable;